run.go

```diff
@@ -150,6 +156,17 @@ var commandRun = cli.Command{
 			Usage:  "output metrics to an external data store (format: type=uri)",
 			EnvVar: "K6_OUT",
 		},
+		cli.StringFlag{
+			Name: "summary-file",
+			Usage: "file where the summary is written (default: standard output)",
+			EnvVar: "K6_SUMMARY_FILE",
+		},
+		cli.StringFlag{
+			Name: "summary-format",
+			Usage: "file format for summary, one of: text, json",
+			EnvVar: "K6_SUMMARY_FILE",
+			Value: "text",
```

Comment:
Let's default to JSON if a summary file is given instead of making it mutually exclusive with the text summary like this. Or YAML, if you prefer that.